python/pyspark/ml/evaluation.py

```diff
@@ -66,6 +66,9 @@ def evaluate(self, dataset, params=None):
         else:
             raise ValueError("Params must be a param map but got %s." % type(params))
 
+    def isLargerBetter(self):
```

Comment:
Can you please copy the doc from Scala here?  (no need to copy to child classes since the "inherit_doc" tag will handle that)